dosomething/fastly-frontend/main.tf

```diff
@@ -29,14 +42,12 @@ resource "fastly_service_v1" "frontend" {
     use_ssl           = true
     port              = 443
   }
-
```

Comment:
This is so funny! Terraform's formatter seems to get confused by the commented out block above and removes spacing between all subsequent parts of this resource. Doesn't seem to cause any issues besides an ugly diff.